### PR TITLE
[Config] Add code coverage via Codeconv & Scoverage

### PR DESCRIPTION
**Pull request by Lewiky**
Integrate code coverage reports with the CI pipeline via Codeconv and Scoverage

### CODE REVIEW

**Comment by Lewiky** [comment]
Coverage task failed here because it required an sbt plugin that's specified in the `plugins.sbt` file, which hasn't been PRed yet....

**Comment by jaylees14** [comment]
@Lewiky But it's included with this PR right? That's what gets run on the CI..?

File: project/plugins.sbt [hunk 1]
```diff
@@ -0,0 +1 @@
+addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.6.1")
```
Comment (c-harding) [inline project/plugins.sbt]: This lack of line break at the end disturbs me, what if I want to cat the file?

**Comment by c-harding** [comment]
The coverage check error looks weirdly like a syntax error

**Review by c-harding — APPROVED** [review]
Looks good, but is `develop` the branch for which we want to share the code cov of long-term, or would `master` be better? Also worth noting that GitHub shows `develop` when the repo is opened.

**Review by jaylees14 — APPROVED** [review]
LGTM. As discussed, would be worth running on merge as well as during PR. Probably only want to do this for `develop` and `master` though.

**Pull request merged**